config/locales/views/en.yml

```diff
@@ -0,0 +1,47 @@
+en:
+  application:
+    navigation_main:
+      dashboard: "Dashboard"
+    navigation_user:
+      sign_in: "Sign in"
+      sign_out: "Sign out"
+      sign_up: "Sign up"
+      edit_profile: "Edit profile"
+  dashboard:
+    index: 'Dashboard'
+  devise:
+    confirmations:
+      new:
+        title: "Resend confirmation instructions"
+    passwords:
+      new:
+        submit: "Send me reset password instructions"
+      edit:
+        update: "Update"
+    registrations:
+      new:
+        title: "Sign up"
+        legend_html: "Create your <strong>%{app_name}</strong> account"
+      edit:
+        update: "Update"
```

Comment:
@timurvafin, that's true for the most translations. However, for 100% dictionary translations it won't be an issue. Can you imagine the situation where instead of "Save" - "Сохранить" we use "Save" - "Записать в БД"?

Also we usually support languages similar to English, right? They are also similar in terms of morphology. For example, in DE and ES "Save" is used in the same way as in English.